### DISCUSSION TITLE
Same search experience for collection entries as well as for assets?

### DISCUSSION THREAD

**Discussion by Alpenverein** [statamic/cms#7376] [Q&A]
I would love to have the same extended search options for entries as well as I have for collection entries today:
It's possible to perform a fielded search for the entries, which is very helpful. My editors asked me for the same options while searching for assets. But it seems there is only a simple fulltext available.. The search seems to search on the "Title" only, but we have build a more extendet blueprint for our images, with legal-stuff, photographer-info and so on... 
It would be so nice to search for this information aswell..
Any ideas?

**Comment by duncanmcclean** [comment]
Hey!

As this is a feature request, would you be able to open an issue on the [statamic/ideas](https://github.com/statamic/ideas/issues) repository instead?

**Reply by duncanmcclean** [reply]
Nevermind, look at @jasonvarga's reply! 😆

**Comment by jasonvarga** [comment]
You can make a search index, and then add it to your asset container.

```php
// config/statamic/search.php
'indexes' => [
	'assets' => [
	    'driver' => 'local',
	    'searchables' => 'all',
	    'fields' => ['path', 'anotherfield'],
	],
],
```

```yaml
title: My Asset Container
search_index: assets
```

If your asset container has a search index, it'll use that. Otherwise, it'll just search the path which is what you're experiencing.

https://statamic.dev/search

**Reply by Alpenverein** [reply]
@jasonvarga Thank you, defining an own index for the assets is working!
I would love to be able to search exclusively in a defined field, like in collections. 

![Bildschirm­foto 2023-01-19 um 09 08 57](https://user-images.githubusercontent.com/46968707/213388596-00d08c0c-6d6d-44f8-acd0-2d7452353049.png)

But that might be a feature request like @duncanmcclean  said.
